platforms/documentation/docs/src/docs/userguide/releases/upgrading/upgrading_version_8.adoc

```diff
@@ -46,6 +46,28 @@ The embedded Kotlin has been updated to link:https://github.com/JetBrains/kotlin
 
 === Deprecations
 
+[[string_invoke]]
+==== Referencing tasks and domain objects by `"name"()` in Kotlin DSL
+
+In Kotlin DSL, it is possible to reference a task or other domain object by its name using the `"name"()` notation.
+
+There are several ways to look up an element in a container by name:
+```
+tasks {
+    "wrapper"() // 1 - returns TaskProvider<Task>
+    "wrapper"(Wrapper::class) // 2 - returns TaskProvider<Wrapper>
+    "wrapper"(Wrapper::class) { // 3 - configures a task named wrapper of type Wrapper
+    }
+    "wrapper" { // 4 - configures a task named wrapper of type Task
+    }
+}
+```
+
+The first notation is now deprecated and will be removed in Gradle 9.0.
+Instead of using `"name"()` to reference a task or domain object, use `named("name")` or one of the other supported notations.
```

Comment:
This looks great. 
Can you add an example of using `named("name")`. That should wrap up the docs.

Reply:
PTAL again